configuration/testversions.properties

```diff
@@ -0,0 +1,8 @@
+2.17.0.redhat-630187=
```

Comment:
tbh I have no strong opinion on that? actually I see no big difference in pulling it from a file or hardcoding it. If you think hardcoded serves us better atm then we can switch to that.